pipeline/extract.py

```diff
@@ -0,0 +1,198 @@
+'''File used for extracting information from bandcamp's website and API'''
+
+from time import sleep
+import json
+import logging
+import asyncio
+import aiohttp
+import requests as req
+from bs4 import BeautifulSoup
+
+
+BANDCAMP_SALES_URL = "https://bandcamp.com/api/salesfeed/1/get_initial"
+MAX_TIMEOUT_SECONDS = 20
+
+
+def get_sale_data_from_api(site_url: str = BANDCAMP_SALES_URL,
+                           max_timeout: int = MAX_TIMEOUT_SECONDS) -> dict:
+    '''Get content from the specified api endpoint'''
+    sleep(10)
```

Comment:
Whats the purpose of the sleep here? Perhaps it would be useful to have a comment above explaining its inclusion.